api.raml

```diff
@@ -165,20 +165,28 @@ types:
       last_name: string
       idp_username: string
       email: string
-      password_meta?: PasswordMeta
+      auth_type:
+        enum: [ login, reset ]
       hide:
         enum: [ yes, no ]
+      last_login: string
```

Comment:
should this be `last_login_utc`?

Reply:
I opted to drop the `_utc` because the password meta dates don't have that suffix.

Reply:
ok, consistent among api at least.